Documentation/Storage-Configuration/Advanced/ceph-mon-health.md

```diff
@@ -125,6 +125,39 @@ CephCluster CR:
 - `spec.network.Provider` : When updated from being empty to "host", Rook fails over all monitors, configuring them to enable or disable host networking.
 - `spec.network.multiClusterService`: When enabled or disabled, Rook fails over all monitors, configuring them to start (or stop) using service IPs compatible with the multi-cluster service.
 
+## Tracking Mon Endpoints
+
+Rook manages a ConfigMap that contains the current set of Ceph monitor IPs, which Ceph clients can use to retrieve mon endpoint data.
```

Comment:
We have always considered the mon endpoints configmap an internal detail and have not documented it. While I don't expect it to ever change, what if we remove lines 130-135 from this doc and just focus on documenting the new endpoints resource?